src/viewer.js

```diff
@@ -426,6 +426,9 @@ $.Viewer = function( options ) {
         }
     }
 
+    // Add updatePixelDensityRatio to resize event
+    $.addEvent( window, 'resize', this.updatePixelDensityRatio.bind(this) );
```

Comment:
We'll need a corresponding `$.removeEvent` in the `destroy` function to properly clean up.

Reply:
I agree. I had to refactor a little to make it work.